docs/uploader/aws-s3-multipart.mdx

```diff
@@ -410,8 +410,83 @@ uppy.use(AwsS3Multipart, {
 });
 ```
 
+#### `getTemporarySecurityCredentials(options)`
+
+:::note
+
+When using [Companion][companion docs] as a backend, you can pass `true` instead
```

Comment:
```suggestion
If you're already using [Companion][companion docs] for remote files, you can pass `true.
It is not recommended to introduce Companion just for S3 signing, use your own backend instead.
```

Reply:
I've copied the wording from https://github.com/transloadit/uppy.io/blob/d8ac350c76175be0df47e009afdf1d82a47dc455/docs/uploader/aws-s3.mdx#L322-L323
I get what you mean, but I think the wording in your suggestion can be a bit confusing, I'd rather stick to the current wording.

Reply:
You referenced example is about not defining this option, for which I agree. But this is passing a boolean instead of implementing a function yourself. I think it's still good to reiterate that you shouldn't introduce Companion to make it 'easier'.